.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           # sleep 10 is needed to allow the st2-docker test containers time to start up.
           # Otherwise st2api is not ready when the tests run.
           sleep 10
-          docker-compose -f ~/st2-docker/docker-compose.yml exec stackstorm st2 action-alias list
+          docker-compose -f ~/st2-docker/docker-compose.yml exec st2client st2 action-alias list
```

Comment:
If that leads you into the rabbit hole, it's possible to just rely on a previous `st2-docker` version from `DEPRECATED/all-in-one` branch: https://github.com/StackStorm/st2-docker/tree/DEPRECATED/all-in-one
It's OK to shelf this work to the later time.

Reply:
Thanks for this, that fixed it (for now). We'll have to circle back around and get it working with the `master` branch of st2-docker. Captured in issue #157.